File: Wordplay/js/5.js
// What are all of the words that have no E or A and are at least 15 letters long?

let main = () => {

    const fs = require("fs");
    let buffer_arry = fs.readFileSync("Wordplay/sowpods.txt", "utf-8");
    let in_file = buffer_arry.split("\n");

    for(let i = 0; i < in_file.length; i++) {

        // First check is word is long enough to meet specification
        if(in_file[i].length > 14) {
            
            if(in_file[i].includes("E") == false &&
            in_file[i].includes("A") == false) {

                console.log(in_file[i])
            };
        };
    };
};

main();

File: Wordplay/python/5.py
# What are all of the words that have no E or A and are at least 15 letters long?
def main():
    in_file = open("../sowpods.txt", "r")
    for words in in_file:
        if("E" not in words and "A" not in words and len(words) > 15):
            print(words, end="")

if __name__ == '__main__':
    main()